src/gen9_avc_encoder.c

```diff
@@ -1161,13 +1176,13 @@ gen9_avc_run_kernel_media_object(VADriverContextP ctx,
     mi_store_data_imm.bo = status_buffer->bo;
     mi_store_data_imm.offset = status_buffer->media_index_offset;
     mi_store_data_imm.dw0 = media_function;
-    gen8_gpe_mi_store_data_imm(ctx, batch, &mi_store_data_imm);
+    gpe->mi_store_data_imm(ctx, batch, &mi_store_data_imm);
```

Comment:
The above is to use the generic structure for the GPE context.
If so, it will be better that the code refine and KBL support are split into two patches.